api/pyproject.toml

```diff
@@ -37,7 +37,7 @@ use_parentheses = true
 multi_line_output = 3
 include_trailing_comma = true
 line_length = 79
-known_first_party=['analytics','app','custom_auth','environments','integrations','organisations','projects','segments','tests','users','webhooks','api','audit','e2etests','features','permissions','util']
+known_first_party=['analytics','app','custom_auth','environments','integrations','organisations','projects','segments','tests','users','webhooks','api','audit','e2etests','features','permissions','task_processor','util']
```

Comment:
Just for the lean diff?

Reply:
Well, partly, but I also thought it logically made sense. It's still 'first party' in my mind. But I did switch back and forth on this so I could definitely be persuaded that it's 'third party'.

Reply:
Why isn't workflows_logic first party then?

Reply:
Good question. IMO it should be. At the moment it's not defined explicitly as either so isort is defaulting to third party, but I'd argue that's incorrect.

Reply:
I don't agree. 1st vs 3rd party import means package local to the codebase vs external dependency. IMO shouldn't add any more meaning to import order, it will only confuse people.

Reply:
Fair enough. I'll revert the latest commit then.

Reply:
Sorry, I thought I'd done this. I have now...